packages/amplication-server/src/core/serviceSettings/serviceSettings.service.ts

```diff
@@ -163,6 +163,8 @@ export class ServiceSettingsService {
             },
           },
           ...settings,
+          authProvider:
+            serviceSettings.authProvider || EnumAuthProviderType.Jwt,
```

Comment:
@morhag90 @gitstart 
When I choose: skip authentication, I see: Authentication: HTTP
I shouldn't see anything

![image](https://github.com/amplication/amplication/assets/39680385/b7397097-b130-4d23-9b33-8e4f9ab9aef0)

Reply:
@abrl91 authProvider is not nullable, so it's either HTTP or JWT. Should we make it nullable or JWT by default?. Also, there's is no option to select HTTP on the service creation wizard currently

Reply:
I understand, but it doesn't make sense to see the authentication section when you choose not to use it. 
@GreenMachine01 @mulygottlieb

Reply:
@abrl91 you are correct. for a service without Auth the "Authentication" should indicate properly the situation.
I think instead of leaving it empty it will be better to show a text like "Not in Use" or "Disabled"